en/services/timesync_v1.md

```diff
@@ -0,0 +1,45 @@
+# Time Synchronization Protocol v1
+
+This protocol is used to estimate the time offsets between system clocks on two MAVLink systems, in a two-system MAVLink network.
+This is useful because it allows ....
+
+The protocol uses just one message [TIMESYNC](#TIMESYNC), which has two `int64_t` fields, `tc1` and `ts1`. 
+The message is sent out with a timestamp in `ts1` (current unix timestamp *1000).
```

Comment:
unix timestamps are in seconds.  ArduPilot uses nanoseconds in this field, so for AP this should be
```suggestion
The message is sent out with a timestamp in `ts1` (current unix timestamp *1e9).
```

Why is it nanoseconds?  Good question.  I may have found something out there that indicated that was the relevant scale.

Reply:
Thanks @peterbarker . 

@julianoes Peter is right that that unix timestamp units are in seconds. The TIMESYNC message doesn't specify a unit (and should).  I think that PX4 essentially has the same interpretation - from the RTPS code where `hrt_absolute_time` is in us - this is the same right?

```
@(topic)_data.tc1 = hrt_absolute_time() * 1000ULL;
```

Reply:
Yes, from what I can see PX4 also uses nanoseconds, phew!